civihr_employee_portal/civihr_employee_portal.module

```diff
@@ -5585,11 +5585,31 @@ function civihr_employee_portal_form_views_exposed_form_alter(&$form, &$form_sta
       _civihr_employee_portal_form_view_exposed_form_alter_actions_wrapper($form, $form_state, $form_id);
       _civihr_employee_portal_form_view_exposed_form_alter_labels_as_placeholders($form, $form_state, $form_id);
       break;
+    case 'views-exposed-form-users-list-page':
+      _civihr_employee_portal_form_view_exposed_alter_users_list($form, $form_state, $form_id);
```

Comment:
Seems the function name is supposed to be `_civihr_employee_portal_form_view_exposed_form_alter_users_list` following the format of the other function names.